Source/Core/DolphinQt/MainWindow.ui

```diff
@@ -159,14 +165,6 @@
     <string>Stop</string>
    </property>
   </action>
-  <action name="actionAboutQt">
-   <property name="text">
-    <string>About &amp;Qt</string>
-   </property>
-   <property name="menuRole">
-    <enum>QAction::AboutQtRole</enum>
```

Comment:
I've Googled some and haven't found any evidence to suggest this. The only serious requirement for LGPL Qt projects I've seen is binaries must dynamically link to Qt.